crate_universe/src/rendering.rs

```diff
@@ -124,6 +138,26 @@ impl Renderer {
         let header = self.engine.render_header()?;
         starlark.push(Starlark::Verbatim(header));
 
+        // Load any `alias_rule`s.
+        let mut loads: BTreeMap<String, BTreeSet<String>> = BTreeMap::new();
+        for alias_rule in Iterator::chain(
+            std::iter::once(&self.alias_rule),
+            context
+                .workspace_member_deps()
+                .iter()
+                .flat_map(|dep| &context.crates[&dep.id].alias_rule),
+        ) {
+            if let Some(bzl) = alias_rule.bzl() {
+                loads.entry(bzl).or_default().insert(alias_rule.rule());
+            }
+        }
+        for (bzl, items) in loads {
+            starlark.push(Starlark::Load(Load {
+                bzl: bzl,
+                items: items,
```

Comment:
We should hard error if any of these alias rules are literally called `"alias"`, as it will conflict with the fact that we reference `alias` below at line 243.